tell_me_your_secrets/__main__.py

```diff
@@ -35,6 +35,7 @@
 arguement_parser.add_argument('-c','--config',help='Path To Another config.yml for Extracting The Data')
 arguement_parser.add_argument('-w','--write',help='Path of the csv File to which results are written')
 arguement_parser.add_argument('-f','--filter',help='Filter the Signatures you want to apply. ',nargs='+')
+arguement_parser.add_argument('-e','--exit',help='Exit non-zero on results found. ',action='store_true')
```

Comment:
Not sure on the best naming for this - open to suggestions.

Reply:
Refactored this code into v1.76. I can reopen if required.

Reply:
What do you mean by that? Ahh I can see the changes in https://github.com/valayDave/tell-me-your-secrets/commit/c116019fd38dcc4d2fe3f48f2b873aff52b40230

However can't see them on the `master` branch?

Reply:
I see it now - wonder how I missed it :/